ocdaction/tasks/models.py

```diff
@@ -12,11 +13,16 @@ class Task(models.Model):
     task_fears = models.CharField(max_length=300, blank=True)
     task_compulsions = models.CharField(max_length=300, blank=True)
```

Comment:
Might need to add `widget=forms.Textarea)` since the field is not showing up as a textarea.